app/views/play/cinematic/Command/AbstractCommand.js

```diff
@@ -103,3 +104,54 @@ export class Noop extends AbstractCommand {
     return Promise.resolve()
   }
 }
+
+/**
+ * AnimeCommand is used to turn animejs animation tweens into commands that the Command Runner can play and cancel.
+ */
+export class AnimeCommand extends AbstractCommand {
```

Comment:
Why did we move this from LankBoss to here?

Reply:
Because we're able to reuse this Command for the dialogSystem.